packages/shared/lib/typings/notification.ts

```diff
@@ -0,0 +1,19 @@
+export type NotificationType = 'info' | 'warning' | 'error'
+
+export type NotificationAction = {
+    label: string
+    isPrimary?: boolean
+    callback?: (notificationData: NotificationData, actionIndex: number) => void
+}
+
+export type NotificationData = {
+    type: NotificationType
+    message: string
+    progress?: number
+    subMessage?: string
+    actions?: NotificationAction[]
+    id?: string
+    ts?: number
```

Comment:
What is `ts`, sorry?

Reply:
I am honestly not sure... I didn't put it there, but just copied the type code over to the new file.